resources/content/schema/base/container.json

```diff
@@ -235,6 +235,14 @@
          "startCount" : {
             "type" : "int",
             "nullable" : true
+        },
+        "blkioDeviceReadIOps": {
+            "type": "array[map[string]]",
```

Comment:
Docker define this way array of map. Should we keep this way or create our own way and then convert it back to docker way at python agent ?